src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -318,7 +405,19 @@ public boolean getTaJunitChkBx() {
         public boolean getTaCoberturaChkBx() {
             return taCoberturaChkBx;
         }
+                
+        public boolean getTaPDFReportChkBx() {
```

Comment:
Why do we need this getter? Can we not access the property directly?

Reply:
Yes, we cannot access these properties directly as they are in a static nested class. To access this in the parent class (MATLAB Builder), getters are required.